packages/cloud_firestore/cloud_firestore_web/lib/cloud_firestore_web.dart

```diff
@@ -125,43 +109,29 @@ class FirebaseFirestoreWeb extends FirebaseFirestorePlatform {
     }
 
     if (settings.host != null && settings.sslEnabled != null) {
-      _webFirestore.settings(firestore_interop.Settings(
+      guardSync(() => _webFirestore.settings(firestore_interop.Settings(
```

Comment:
`settings` apparently did not throw a FirebaseException before. Is this a voluntary behavior change?